catalog-info.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: bluesky-worker
+  title: bluesky-worker
+  description: Lightweight wrapper around bluesky services
+  annotations:
+    github.com/project-slug: DiamondLightSouce/blueapi
+spec:
+  type: service
+  lifecycle: production
+  owner: user:vid18871 # TODO: owner: DAQ-Core
+  #  system: Athena  # TODO: Define Athena system: presumably same location as DAQ-Core/DAQ?
+  providesApis:
+    - message-topics
+    - restapi
+    - blueskydocument-to-scanmessage
```

Comment:
I think the ugliness of this (since it's not actually emitting ScanMessages, and ScanMessages are defined elsewhere, and this whole provided API is defined elsewhere) is reason enough to say: now is the time to do #167  and fold it into this change.

Reply:
I would prefer to address that in a separate change that also includes updating the AsyncAPI definition to reflect that some of it's functionality is now repaced by the REST API.  That will allow us to close this ticket and its associate Jira one and schedule the cleanup activity for next sprint.